### PR TITLE
feat(admin-ui): add info element to metrics

### PR DESCRIPTION
**Pull request by mgs95**

### CODE REVIEW

File: admin/admin-ui/src/Pages/Version/pages/Metrics/boxes/ConfusionMatrixBox/ConfusionMatrixBox.tsx [hunk 1]
```diff
@@ -28,6 +29,12 @@ function ConfusionMatrixBox({ toggleExpanded, nodeId, data }: Props) {
   return (
     <Box>
       <Title text="Confusion Matrix" />
+      <Info>
```
Comment (guumaster) [inline admin/admin-ui/src/Pages/Version/pages/Metrics/boxes/ConfusionMatrixBox/ConfusionMatrixBox.tsx:32]: What if we move all Info object for every component to a single file to maintain them easier?

Comment (mgs95) [reply admin/admin-ui/src/Pages/Version/pages/Metrics/boxes/ConfusionMatrixBox/ConfusionMatrixBox.tsx:32]: Do you mean the content of the info?

Comment (guumaster) [reply admin/admin-ui/src/Pages/Version/pages/Metrics/boxes/ConfusionMatrixBox/ConfusionMatrixBox.tsx:32]: yes

Comment (mgs95) [reply admin/admin-ui/src/Pages/Version/pages/Metrics/boxes/ConfusionMatrixBox/ConfusionMatrixBox.tsx:32]: sounds good

**Review by angelsanchez — APPROVED** [review]

**Pull request merged**